test/checks/checks.test.js

```diff
@@ -29,3 +29,10 @@ describe('canSimplifyPolynomialTerms addition', function() {
   ];
   tests.forEach(t => testCanCombine(t[0], t[1]));
 });
+
+describe('canSimplifyPolynomialTerms denominator in numerator', function() {
+  const tests = [
+    ['(2x + 3)/(2x + 2)', true],
```

Comment:
can you add a bunch more tests? especially a cases that should be false

you can use some of my comments above for edge cases to test for too :)